lib/irgen.ml

```diff
@@ -10,27 +10,26 @@ let translate (binds, sfuncs): L.llmodule =
   let f_t = L.double_type context 
   and i1_t = L.i1_type context 
   and i8_t = L.i8_type context 
+  and i32_t = L.i32_type context
```

Comment:
is this used for indexing?

Reply:
yes I use i32's when I cast floats to ints (specifying size of malloc, specifying indexes, etc.). So there's gonna be a limit on the size of arrays but I think that number is big enough.